README.md

```diff
@@ -15,14 +15,9 @@ None.
 
 ## Role Variables ##
 
-None.
-
-<!--
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| optional_variable | Describe its purpose. | `default_value` | No |
-| required_variable | Describe its purpose. | n/a | Yes |
--->
+| backports_update_cache | Indicate if the package cache should be updated after adding the backports repository. | `false` | No |
```

Comment:
I think the default value should be `true`, as that is what users will most frequently desire.  It also matches [the default behavior of the `ansible.builtin.apt_repository` task](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_repository_module.html#parameter-update_cache).

Reply:
See commit [18087a3](https://github.com/cisagov/ansible-role-backports/pull/42/commits/18087a38ba1920f369eb62afbe8efe2432860588).